src/test/java/yanagishima/TestHelper.java

```diff
@@ -0,0 +1,18 @@
+package yanagishima;
+
+import static org.apache.commons.codec.digest.DigestUtils.md5Hex;
+import static yanagishima.util.QueryEngine.elasticsearch;
+
+import lombok.experimental.UtilityClass;
+import yanagishima.util.QueryIdUtil;
+
+@UtilityClass
+public class TestHelper {
+  public static String getQueryId() {
+    return QueryIdUtil.generate("test-datasource", "SELECT 'test'", elasticsearch.name());
+  }
+
+  public static String getPublishId() {
+    return md5Hex("test-datasource" + ";" + "test-engine" + ";" + getQueryId());
```

Comment:
nitpick
`md5Hex("test-datasource" + ";" + elasticsearch.name() + ";" + getQueryId());` is better?
Because `getQueryId` is based on engine=elasticsearch

Reply:
Updated. Let me change to constant.